recipes/default.rb

```diff
@@ -119,7 +119,7 @@
 end
 
 begin
-  kibana_ip = private_recipe_ip("hopslog","default")
+  kibana_ip = public_recipe_ip("hopslog","default")
```

Comment:
What is the reason for this change? We use everywhere the `private_recipe_ip`.

Reply:
Since we are removing the kibana proxy servlet and instead exposing a secure kibana to users.